# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wgpu/.github/pull_request_template.md
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/gfx/wgpu/.github/pull_request_template.md b/gfx/wgpu/.github/pull_request_template.md
deleted file mode 100644
--- a/gfx/wgpu/.github/pull_request_template.md
+++ /dev/null
@@ -1,8 +0,0 @@
-**Connections**
-_Link to the issues addressed by this PR, or dependent PRs in other repositories_
-
-**Description**
-_Describe what problem this is solving, and how it's solved._
-
-**Testing**
-_Explain how this change is tested._